src/passes/stringify-walker-impl.h

```diff
@@ -51,14 +51,7 @@ template<typename SubType>
 inline void StringifyWalker<SubType>::scan(SubType* self, Expression** currp) {
   Expression* curr = *currp;
   if (Properties::isControlFlowStructure(curr)) {
-    Expression* copy = curr;
-    // To ensure control flow children are walked consistently during outlining,
-    // we push a copy of the control flow expression. This avoids an issue where
-    // control flow no longer points to the same expression after being
-    // outlined into a new function.
-    self->controlFlowQueue.push(copy);
-    DBG(std::cerr << "controlFlowQueue.push: " << ShallowExpression{copy}
-                  << ", " << copy << "\n");
```

Comment:
We could still do this logging with `curr` instead of `copy` if you wanted, but lgtm either way.